plugin.xml

```diff
@@ -7,8 +7,8 @@
         <repo>https://github.com/Microsoft/cordova-plugin-code-push.git</repo>
 
         <dependency id="code-push" version=">=1.8.0-beta" />
-        <dependency id="cordova-plugin-file-transfer" version=">=1.3.0" />
-        <dependency id="cordova-plugin-file" version=">=3.0.0" />
+        <dependency id="cordova-plugin-file" url="https://github.com/apache/cordova-plugin-file.git" commit="1e2593f42455aa78d7fff7400a834beb37a0683c" /> 
```

Comment:
Did you talk to the Cordova guys by any chance about this issue? When are they planning a new release of the file plugin?